code/modules/psionics/abilities/grip.dm

```diff
@@ -29,19 +29,21 @@
 		return
 	if(world.time < next_squeeze_time)
 		return
-	if(!length(get_line(user, victim)))
+	var/rangecheck = length(get_line(user, victim))
+	if(!length(get_line(user, victim)) || rangecheck > 8)
 		to_chat(user, SPAN_WARNING("You need to have direct line of sight to your target!"))
 		return
 	. = ..()
 	if(!.)
 		return
-	user.visible_message(SPAN_WARNING("[user] squeezes [user.get_pronoun("his")] hand!"), SPAN_WARNING("You squeeze your hand to tighten the psionic force around [victim]."))
-	log_and_message_admins("[key_name(owner)] has psionically crushed [victim]", owner, get_turf(owner))
-	to_chat(victim, SPAN_DANGER(FONT_HUGE("You are crushed by an invisible force!")))
-	victim.apply_damage(20, DAMAGE_BRUTE, armor_pen = 15, def_zone = BP_HEAD)
-	victim.SetStunned(2)
-	apply_cooldown(user)
-	next_squeeze_time = world.time + 2 SECONDS
+	if (rangecheck < 9)
```

Comment:
That check can/should be removed as the range is already checked above.

Reply:
Done